.github/workflows/continuous-integration.yml

```diff
@@ -73,10 +70,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - name: Use Node.js 14
-        uses: actions/setup-node@v2
-        with:
-          node-version: 14
+      - uses: actions/setup-node@v2
```

Comment:
Explicitly defining a version allows for a more robust CI pipeline (i.e. It doesn't break because a new LTS version is released).

Reply:
Ok